github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml

```diff
@@ -771,3 +771,56 @@ periodics:
       resources:
         requests:
           memory: "1Gi"
+- annotations:
+    testgrid-create-test-group: "false"
+  cluster: ibm-prow-jobs
+  decorate: true
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  extra_refs:
+  - org: kubevirt
+    repo: project-infra
+    base_ref: main
+    workdir: true
+  interval: 168h
+  labels:
+    preset-bazel-cache: "true"
+    preset-gcs-credentials: "true"
+    preset-github-credentials: "true"
+  max_concurrency: 1
+  name: periodic-project-infra-perf-report-creator
+  spec:
+    containers:
+    - image: quay.io/kubevirtci/golang:v20230502-bfaa042
+      env:
+        - name: GIMME_GO_VERSION
+          value: "1.19"
+        - name: GIT_AUTHOR_NAME
+          value: kubevirt-bot
+        - name: GIT_AUTHOR_EMAIL
+          value: kubevirtbot@redhat.com
+        - name: TARGET_GITHUB_REPO
+          value: kubevirt/ci-performance-benchmarks
+      command:
+        - "/usr/local/bin/runner.sh"
+        - "/bin/sh"
+        - "-ce"
+        - |
+          (
+            cd ..
+            git clone "https://kubevirt-bot@github.com/$TARGET_GITHUB_REPO.git"
+            cd ci-performance-benchmarks
+            git config user.email "$GIT_AUTHOR_EMAIL"
+            git config user.name "$GIT_AUTHOR_NAME"
+          )
+          ci_perf_repo_dir=$(cd ../ci-performance-benchmarks && pwd)
+          bazel run //robots/cmd/perf-report-creator -- results --credentials-file $GOOGLE_APPLICATION_CREDENTIALS --output-dir $ci_perf_repo_dir/output/results
```

Comment:
I think we need to add `--since 168h0m0s` to the flags to make sure that all jobs are scraped. The current default is 24 hours
```
  -since duration
        Filter the periodic job in the time window (default 24h0m0s)
```

Reply:
We can also run it 24h if you want. Whatever makes more sense.